src/trec_search.cc

```diff
@@ -159,7 +159,7 @@ int main(int argc, char **argv)
 	else if (config.use_weightingscheme("lmweight")) {
 		cout<<"Config Check LM"<<config.check_lmweight()<<endl;
 		if (config.check_lmweight()) {
-			enquire.set_weighting_scheme(Xapian::LMWeight(config.get_lmparam_log(),config.get_lmparam_select_smoothing(),config.get_lmparam_smoothing1(),config.get_lmparam_smoothing2()));
+			enquire.set_weighting_scheme(Xapian::LMWeight(config.get_lmparam_log(),config.get_lmparam_select_smoothing(),config.get_lmparam_smoothing1(),config.get_lmparam_smoothing2(),config.get_lmparam_delta(),config.get_lmparam_enable_dirplus()));
```

Comment:
Dir+ is basically the modified Dirichlet prior method of Language model. Thus, it was implemented as a part of it by passing couple more required parameters. Please refer to this PR: https://github.com/xapian/xapian/pull/111 to grab Dir+ changes -- module should compile with these changes. Also, in the future when this PR gets merged then this LMWeight execution should compile fine.